packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/test/lib_pin_code/blocs/update_and_verify_pin_bloc_test.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:local_session_timeout/local_session_timeout.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rx_bloc_test/rx_bloc_test.dart';
+import 'package:{{project_name}}/base/common_blocs/coordinator_bloc.dart';
+import 'package:{{project_name}}/lib_pin_code/bloc/update_and_verify_pin_bloc.dart';
+import 'package:{{project_name}}/lib_pin_code/services/pin_biometrics_service.dart';
+import 'package:{{project_name}}/lib_pin_code/services/verify_pin_code_service.dart';
+
+import '../../base/common_blocs/coordinator_bloc_mock.dart';
+import '../mock/pin_biometrics_service_mock.dart';
+import '../mock/verify_pin_code_service_mock.dart';
+import '../stubs.dart';
+
+void main() {
+  late CoordinatorBlocType coordinatorBloc;
+  late VerifyPinCodeService verifyPinCodeService;
+  late PinBiometricsService pinBiometricsService;
+  late CoordinatorStates coordinatorStates;
+  late CoordinatorEvents coordinatorEvents;
+
+  void defineWhen({
+    String? pinCode,
+    String? encryptedPinCode,
+    bool isPinCorrect = false,
+    bool isPinCreated = false,
+    bool isPinCodeInSecureStorage = false,
+    bool isPinDeleted = false,
+  }) {
+    when(coordinatorStates.userLoggedIn)
+        .thenAnswer((_) => const Stream.empty());
+
+    when(coordinatorStates.userLogOut)
+        .thenAnswer((_) => Stream.value(null));
+
+    when(verifyPinCodeService.deleteStoredPin())
+        .thenAnswer((_) => Future.value());
+
+    when(pinBiometricsService.areBiometricsEnabled())
+        .thenAnswer((_) => Future.value(true));
+
+    when(pinBiometricsService.setBiometricsEnabled(true))
+        .thenAnswer((_) => Future.value());
+
+    when(pinBiometricsService.setBiometricsEnabled(false))
+        .thenAnswer((_) => Future.value());
+
+    /*when(verifyPinCodeService.setPinCodeType(false))
+        .thenAnswer((_) => Future.value(false));
+
+    when(verifyPinCodeService.setPinCodeType(true))
```

Comment:
Same as above

Reply:
This should be fixed now. Can you review again and resolve the conversation if you think it's fixed.